package.json

```diff
@@ -4,8 +4,10 @@
   "description": "",
   "main": "index.jsx",
   "scripts": {
-    "build": "webpack --config webpack.prod.js",
-    "start:dev": "webpack-dev-server --inline --hot --config webpack.dev.js",
+    "build": " webpack --config webpack.prod.js",
```

Comment:
A question more than anything, can you add webpack aliases? I'd assume you could, I haven't tried with node though. It's just be tidier with your routes :)

Reply:
It's a different process for Node and I've tried a couple of different ways @girwin90 without much success :(. I'll create another ticket for it in the backlog and investigate it separately from this PR.

Reply:
There are some nice approaches for aliases for Node. I can give you a shout on this Fiona